tests/dummy/app/templates/demos/sl-date-time.hbs

```diff
@@ -69,19 +69,19 @@
 
 <h3>Properties</h3>
 <div class="list-group">
-    {{#property-text name="timezone" type="string" required=true}}
+    {{#property-text name="timezone" type="String" required=true}}
         The string value for the current timezone name (i.e., "America/Chicago").
     {{/property-text}}
 
-    {{#property-text name="format" type="string" default="\"datetime\""}}
+    {{#property-text name="format" type="String" default="\"datetime\""}}
         The string format to show the datetime text in. Can be "date", "datetime", or "relative".
     {{/property-text}}
 
-    {{#property-text name="locale" type="string" default="\"en\""}}
+    {{#property-text name="locale" type="String" default="\"en\""}}
         The locale string to use for <em>moment</em> date formats.
     {{/property-text}}
 
-    {{#property-text name="value" type="mixed" default="now"}}
+    {{#property-text name="value" type="Date" default="now"}}
```

Comment:
This should be `*` as https://github.com/softlayer/sl-ember-components/blob/develop/addon/components/sl-date-time.js#L156 runs this value through moment, which can accept a variety of formats http://momentjs.com/docs/#/parsing/ which is mentioned in description of this property on line 85.